src/main/windows/mainWindow.ts

```diff
@@ -324,6 +324,12 @@ export class MainWindow extends EventEmitter {
     private onFocus = () => {
         // Only add shortcuts when window is in focus
         if (process.platform === 'linux') {
+            const isKDE = (process.env.XDG_CURRENT_DESKTOP ?? '').toUpperCase() === 'KDE' ||
```

Comment:
Two things here:
1) Did we check what happens here when the window is restored from the taskbar, or when its just focused normally? I'm a bit hesitant to always be escaping the `onFocus()` function on KDE

2) Nice check for the KDE environment variable! Can we toss that in `main/utils.ts` so we can use it in more than one place?

Reply:
1) yes, see attached video (if there is something else you would like me to verify, please comment and I will check, but so far, it seems to be working fine in all scenarios and only does the escape when is minimized)

2) done in my last commit.


https://github.com/user-attachments/assets/5162107d-b309-4df5-b1df-7db08c8d4cff